terraform/Makefile

```diff
@@ -29,9 +24,11 @@ clean: tfclean
 .PHONY: build
 build: clean
 	GOOS=$(OS) GOARCH=$(ARCH) $(CGOFLAG) go build -o $(BUILDDIR)/terraform-provider-teleport $(BUILDFLAGS)
-	mkdir -p $(PROVIDER_PATH)
-	mv $(BUILDDIR)/terraform-provider-teleport $(PROVIDER_PATH)
+
+.PHONY: apply
+apply: install
+	cd $(TFDIR) && terraform init -var-file="vars.tfvars" && terraform apply -auto-approve -var-file="vars.tfvars" 
```

Comment:
Nit: Unless I'm mistaken, doing `cd` in a makefile target will leave you in that directory so it becomes hard to chain multiple targets. I would just pass proper paths to terraform command.